connectors/email/src/main/java/io/camunda/connector/email/outbound/protocols/actions/ContentType.java

```diff
@@ -7,11 +7,15 @@
 package io.camunda.connector.email.outbound.protocols.actions;
 
 import com.fasterxml.jackson.annotation.JsonEnumDefaultValue;
+import io.camunda.connector.generator.java.annotation.DropdownItem;
 
 public enum ContentType {
   @JsonEnumDefaultValue
+  @DropdownItem(label = "PLAIN", order = 0)
   PLAIN("text/plain; charset=utf-8"),
+  @DropdownItem(label = "HTML", order = 1)
   HTML("text/html; charset=utf-8"),
+  @DropdownItem(label = "HTML & Plaintext", order = 2)
   MULTIPART("multipart/mixed; charset=utf-8");
```

Comment:
Lets use the order returned by `Enum.values()` instead of requiring to define it.